package.json

```diff
@@ -61,9 +65,7 @@
     "mocha": "^2.2.5",
     "phantomjs": "^1.9.17",
     "react-hot-loader": "^1.2.8",
-    "react-router": "^0.13.3",
-    "react-stub-context": "^0.3.0",
```

Comment:
yeh `react-stub-context` hasn't been updated yet to react 0.14.
@shaurya947 said to just leave it out if it isn't used by the files source files : https://github.com/callemall/material-ui/pull/1704